opba-protocols/protocol-bpmn-shared/src/main/java/de/adorsys/opba/protocol/bpmnshared/config/flowable/SerializerUtil.java

```diff
@@ -2,6 +2,7 @@
 
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
+// import com.google.common.collect.ImmutableMap;
```

Comment:
commenting import `import com.google.common.collect.ImmutableMap;` 
and then 
importing it again on next line makes no sense

Reply:
funny ;-)


> Dockerfile's should be updated as well:
> i.e. opba-embedded-starter/Dockerfile
> `FROM openjdk:8-jre-alpine` is Java 8

Yep, thanks, I changed to dockerfiles too. I did not test with docker but with launching everything locally with intellij. And it works fine. Including local xs2a sandbox.